.env.example

```diff
@@ -6,6 +6,7 @@ PG_CONN="postgres://postgres:password@localhost:5432/archive"
 ENABLE_GRAPHIQL="true"
```

Comment:
You can rework Readme section with `For an example of some sensible defaults, see below:` since now you provide `.env.example`.

Reply:
Sounds good! I will update the README in another PR when all changes have been made so I don't have to keep editing it :D